integration/tests/cook/test_basic.py

```diff
@@ -720,7 +724,7 @@ def test_memory_limit_exceeded_mesos_script(self):
     def test_get_job(self):
         # schedule a job
         job_spec = util.minimal_job()
-        resp = util.session.post('%s/rawscheduler' % self.cook_url, json={'jobs': [job_spec]})
+        resp = util.session.post('%s/jobs' % self.cook_url, json={'jobs': [job_spec], 'pool' : util.default_submit_pool()})
```

Comment:
We should just do:

```suggestion
        _, resp = util.submit_job(self.cook_url)
```

`submit_job` takes care of using the correct endpoint, submitting a `minimal_job()`, and using the `default_submit_pool()`.

Reply:
Nice.